### PR TITLE
Feature/aws s3 upload

### PR DESCRIPTION
**Pull request by pistoolster**
1.add env file template
2.pictures now can upload to aws s3

### CODE REVIEW

File: env_template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# cp env_template .env
+# all variables in .env file will be loaded into environment variables after application starts
+[base]
```
Comment (kunkun-tang) [inline env_template]: [base] is not a comment line, and python could recognize it?

File: app/complaint_api.py [hunk 2]
```diff
@@ -71,7 +73,8 @@ def post(self):
             pic_file = '%s%s' % (destination, args['sequence'] + '.jpeg')
```
Comment (kunkun-tang) [inline app/complaint_api.py]: Discussed with @e-cloud, we need to refactor this API as well. Basically, we should consolidate upload_id and upload_invoice to one API, for example, upload_file. The new API accepts two parameters, first is the type of file (id, or evidence picture), and the second is pic_file. In other words, `sequence` is not necessary any longer.

File: app/aws/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"""
+desc: 
+file: __init__.py.py
```
Comment (kunkun-tang) [inline app/aws/__init__.py]: I like the practice to always have an __init__.py in every module. I know it is required for python2, and python3 allows modules don't come with it. Just curious what are more advantages of __init__.py?

File: env_template (hunk 1 above)
Comment (pistoolster) [reply env_template]: that's all right, it's .env/ini file grammar，just mark a section.

File: app/complaint_api.py [hunk 4]
```diff
@@ -36,50 +38,26 @@ def post(self):
         print(session)
         user_id = session["user_id"]
         args = file_upload.parse_args()
+        if args['upload_type'] not in ['invoice', 'id']:
```
Comment (kunkun-tang) [inline app/complaint_api.py]: 3 types in total. Also need `evidence`

Comment (pistoolster) [reply app/complaint_api.py]: added

**Pull request merged**